lib/Elevate/Components/MySQL.pm

```diff
@@ -72,6 +72,17 @@ sub _reinstall_mysql_packages {
     INFO("Restoring $cnf_file.rpmsave_pre_elevate to $cnf_file...");
     File::Copy::copy( "$cnf_file.rpmsave_pre_elevate", $cnf_file ) or WARN("Couldn't restore $cnf_file.rpmsave: $!");
 
+    # Apparently the APITool is in fact unable to figure out that --output=json means I want JSON in this context,
+    # so just load the modules directly and do what the API does to gather the data I want.
+    require Cpanel::Services::Enabled;
+    my $enabled = Cpanel::Services::Enabled::is_enabled('mysql');
+    if( !$enabled ) {
+        INFO("MySQL is not enabled. This will cause the MySQL upgrade tool to fail. Temporarily enabling it to ensure the upgrade succeeds.");
+        Cpanel::SafeRun::Simple::saferunnoerror( qw{/usr/local/cpanel/bin/whmapi1 configureservice service=mysql enabled=1} );
+        # Pray it goes ok, as what exactly do you want me to do if this reports failure? May as well just move forward in this case without checking.
+
+    }
+
```

Comment:
what if the elevate process crash and we have to run `--continue` from there, we will use an incorrect state